packages/breadboard/breadboard.schema.json

```diff
@@ -10,16 +10,42 @@
     }
   },
   "type": "object",
-  "required": ["nodes", "edges"],
+  "required": [
+    "nodes",
+    "edges"
+  ],
   "additionalProperties": false,
   "properties": {
+    "$schema": {
+      "type": "string",
+      "format": [
+        "uri",
+        "uri-reference"
+      ]
+    },
+    "title": {
+      "description": "A human-readable title for the graph",
+      "type": "string"
+    },
+    "description": {
+      "description": "",
+      "type": "string"
+    },
+    "version": {
+      "description": "A version string for the graph",
+      "type": "string",
+      "pattern": "^(0|[1-9]\\d*)\\.(0|[1-9]\\d*)\\.(0|[1-9]\\d*)(?:-((?:0|[1-9]\\d*|\\d*[a-zA-Z-][0-9a-zA-Z-]*)(?:\\.(?:0|[1-9]\\d*|\\d*[a-zA-Z-][0-9a-zA-Z-]*))*))?(?:\\+([0-9a-zA-Z-]+(?:\\.[0-9a-zA-Z-]+)*))?$"
```

Comment:
This is semver, I assume? Maybe just mention that in the description

Reply:
I even thought about updating it, then got distracted by the changesets